index.html

```diff
@@ -134,7 +134,7 @@ <h2>Software Developer</h2>
             </div>
```

Comment:
- [ ] For all the images for the social media icons, add the necessary alt text giving the description for the respective image.